content/06.discussion-a.md

```diff
@@ -1,82 +1,69 @@
-
-Moreover. It seems there are sometimes unwritten roles that need to be explored in a way to learn and use GitHub and that in such Github collaborations, practice and redo practice play an important role. For instance, in one case during cloning the project to work on it, there will be unknown conflicts and limitations for downloading and cloning process on local computers because of low internet speed. But it seems it would be much easier if we choose "open with GitHub desktop" instead of using " Download Zip
+Though GitHub has been available as a platform for more than a decade at time of writing, its uptake among EEB researchers, especially as a tool for collaboration, has been slow.  
+Some research groups choose to take advantage of alternative platforms with similar capabilities, such as GitLab or Bitbucket, but many have yet to integrate these types of project management software into their lab ecosystems at all.  
+Some attribute this hesitation to the steep learning curve combined with limited instruction available through traditional university courses.  
+When use of GitHub is taught within an EEB context, it is usually accompanying coursework in topics such as statistical programming, and some students may find it overwhelming to juggle learning git alongside scripting languages, statistical theory, and file system navigation, especially when many may also be new to using command-line interfaces in general.  
+Instructors likewise may confuse the expected digital literacy of younger students with computational fluency, even when modern technology increasingly abstracts many relevant concepts through search optimization and preponderant IDEs.  
+The default public nature of GitHub usage can add additional pressure to students learning to use the platform (Gomes et al. In Prep).   
+Thus, hesitancy may come from a more general reluctance to share data and code publicly [@doi:10.1038/s41597-021-00981-0], or technical and logistical issues related to storage of large data files and lack of integration with other research platforms.  
 
-### Using GitHub is a good start, but lots of practices to make repo more user friendly
+While many quantitative EEB researchers may take advantage of GitHub for individual use, collaborative use may lag behind due to how researchers traditionally divide labor within projects.  
+GitHub remains a tool predominantly used by computer scientists and software developers, and EEB researchers may view GitHub as a platform that only needs to be used by individuals writing code.      
+Those assumptions may have obscured the utility of GitHub for tasks other than traditional data analysis and software development, or how GitHub can facilitate the integration of code with non-coding aspects of projects through the practice of repository design.  
 
 
-*Contributors to this section: Ali, Emma* 
+### Box 1
```

Comment:
This is Box 2 now, right?

Reply:
Great, thanks! I just updated that to box 2